pkg/cloudprovider/nodecontroller/nodecontroller.go

```diff
@@ -136,6 +137,61 @@ func NewNodeController(
 	}
 }
 
+// Generates num pod CIDRs that could be assigned to nodes.
+func generateCIDRs(num int) util.StringSet {
+	res := util.NewStringSet()
+	for i := 0; i < num; i++ {
+		// TODO: Make the CIDRs configurable.
+		res.Insert(fmt.Sprintf("10.244.%v.0/24", i))
```

Comment:
This part needs to be addressed now that `KUBE_GCE_CLUSTER_CLASS_B` is in. (https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/gce/config-default.sh#L66)